### PR TITLE
BLD: Updated biggus requirements in INSTALL

### LINKED ISSUE #1514: BLD: Bad ref to dependencies
Currently the only manner in which to determine which dependencies to build to get **master** up and running is by looking at the conda build recipes.  The **INSTALL** should be updated to reflect the version requirements of packages.

As a result of https://github.com/SciTools/iris/pull/1424/files, iris master actually requires a version of biggus greater than 0.7 (it requires a version which includes the commit makes API changes to biggus.save https://github.com/SciTools/biggus/commit/89ced476848882cb84616b7d9c2ce2959d7ae294).

I propose that we make a new tag release of biggus and then update the INSTALL to reflect this requirement.

Biggus issue: https://github.com/SciTools/biggus/issues/114

### PR DESCRIPTION
**Pull request by cpelley**
Closes #1514

### CODE REVIEW

**Comment by rhattersley** [comment]
It would make sense to sort out #1519 at the same time...?

**Comment by cpelley** [comment]
Since https://github.com/SciTools/conda-recipes-scitools/pull/47 has now been merged yes

**Comment by cpelley** [comment]
Ready

File: .travis.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ install:
   # ---------------------------------
   - conda config --add channels scitools
   - conda install --quiet --file conda-requirements.txt
-  - conda install --quiet -c scitools/channel/dev biggus=0.7.0.post.180 --no-deps --yes
+  - conda install --quiet -c scitools/channel/dev biggus=0.8.0 --no-deps --yes
```
Comment (esc24) [inline .travis.yml]: I'd have thought we could remove this altogether as its covered by the `conda-requirements.txt` Am I missing something?

Comment (rhattersley) [inline .travis.yml]: > it's covered by the conda-requirements.txt

:+1: This line should be removed and the requirements file should be updated to say biggus>=0.8.

**Comment by pelson** [comment]
:+1: - would you mind squashing?

**Comment by cpelley** [comment]
All done :)

**Pull request merged**